sources/packages/web/src/views/student/StudentProfile.vue

```diff
@@ -126,31 +98,55 @@ export default {
         studentAllInfo.value.pdVerified === null,
     );
 
-    const appliedPDButton = () => {
-      showApplyPDButton.value = false;
-      if (
-        studentAllInfo.value?.validSin &&
-        studentAllInfo.value?.pdSentDate === null &&
-        studentAllInfo.value?.pdVerified === null
-      ) {
-        showApplyPDButton.value = true;
+    const getStudentDetails = async () => {
+      const studentRestriction = await StudentService.shared.getStudentRestriction();
+      hasRestriction.value = studentRestriction.hasRestriction;
+      restrictionMessage.value = studentRestriction.restrictionMessage;
+      if (props.editMode) {
+        await getStudentInfo();
+        const data: StudentFormData = {
+          ...studentAllInfo.value,
+          ...studentAllInfo.value.contact,
```

Comment:
not part of this PR, with lines 108 and 109, data will be something like 
![image](https://user-images.githubusercontent.com/77353155/156443046-9317c1d5-d080-4eb0-83aa-c43de04a6b90.png)
is this the expected result? @andrewsignori-aot  @guru-aot @dheepak-aot

Reply:
As we talked about in the call, this is copying the contact data to the object root because the current form.io definition is expected this strcutuire.
In the future, we can avoid this using the container component from form.io to create a child object.
![image](https://user-images.githubusercontent.com/61259237/156448096-1890225f-f166-4c16-976b-6bf761cb412c.png)